pkg/api/postage.go

```diff
@@ -173,6 +175,14 @@ func (s *Service) postageGetStampsHandler(w http.ResponseWriter, r *http.Request
 			jsonhttp.InternalServerError(w, "unable to check batch")
 			return
 		}
+		if !exists {
+			state := s.batchStore.GetChainState()
+			if state.TotalAmount.Cmp(v.Amount()) == -1 {
+				isExpired = true
+			} else {
+				isExpired = false
```

Comment:
This is not required if you define the variable inside the loop. The default will always be false.